lib/compiler.ts

```diff
@@ -4,11 +4,15 @@ import { path, ts } from "./mod.deps.ts";
 import { ScriptTarget } from "./types.ts";
 
 export function outputDiagnostics(diagnostics: readonly ts.Diagnostic[]) {
-  console.error(ts.formatDiagnosticsWithColorAndContext(diagnostics, {
+  const host: ts.FormatDiagnosticsHost = {
     getCanonicalFileName: (fileName) => path.resolve(fileName),
     getCurrentDirectory: () => Deno.cwd(),
     getNewLine: () => "\n",
-  }));
+  };
+  const output = Deno.env.get("NO_COLOR") == null
+    ? ts.formatDiagnosticsWithColorAndContext(diagnostics, host)
+    : ts.formatDiagnostics(diagnostics, host);
```

Comment:
`Deno.noColor` can be used so that `dnt` doesn't require additional permissions to access the environment variable

```suggestion
  const output = Deno.noColor
    ? ts.formatDiagnostics(diagnostics, host)
    : ts.formatDiagnosticsWithColorAndContext(diagnostics, host);
```

Reply:
Oh, I didn't know about this property. I opened https://github.com/denoland/dnt/pull/223